### PR TITLE
Readme merge from discord text file

### PR DESCRIPTION
**Pull request by cychow**
Adding README.md based off of Jack's ideas text file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+`/azurebot server login <name>` (sends login information to the user if they have permission)
+
+## v4 - agent
+- enlightened agent on the machine can perform server-specific actions
+- turn off as soon as no players are connected
+
+
+
+
```
Comment (jackhorton) [inline README.md]:
```suggestion
## Proposed SQL Schema
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+### Servers
+Id, Name, ResourceId, Game, UserId, GuildId, CurrentState, CurrentSku
+
+### ServerStateLog
+Id, ServerId, Timestamp, NewState, NewSku
+
+
+
+### Custom images
```
Comment (jackhorton) [inline README.md]:
```suggestion
## Custom images
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+## v1 - basic functionality
```
Comment (jackhorton) [inline README.md:3]:
```suggestion
# Roadmap

## v1 - basic functionality
```

**Review by jackhorton — APPROVED** [review]
Looks good overall, just some minor suggestions

**Review by cychow — COMMENTED** [review]
Suggestions seem fine; it was your ideas file in the first place

**Pull request merged**